src/lib/page-state/_core.scss

```diff
@@ -0,0 +1,84 @@
+@use './token-utils' as *;
+@use '../core/styles/typography';
+
+@forward './token-utils';
+
+@mixin host {
+  display: flex;
+  width: 100%;
+  justify-content: center;
+  container-type: inline-size;
+}
+
+@mixin host-hidden {
+  display: none;
+}
+
+@mixin base {
+  width: #{token(width)};
+  margin-top: #{token(spacing)};
+  margin-bottom: #{token(spacing)};
+
+  text-align: center;
+}
+
+@mixin base-mobile {
+  @include override(width, mobile-width);
+}
+
+@mixin graphic {
+  @include override(height, graphic-height);
+  @include override(spacing, graphic-spacing);
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  width: 100%;
+}
+
+@mixin graphic-slotted {
+  height: 100%;
+  width: 100%;
+}
+
+@mixin graphic-mobile {
+  @include override(height, mobile-graphic-height);
+}
+
+@mixin title {
+  @include typography.style(heading8);
+
+  color: #{token(title-color)};
+  margin-top: #{token(title-spacing)};
+  margin-bottom: #{token(title-spacing)};
+}
+
+@mixin title-mobile {
+  @include typography.style(heading5);
+}
+
+@mixin message {
+  @include typography.style(body4);
+
+  color: #{token(message-color)};
+  margin-bottom: #{token(message-spacing)};
+}
+
+@mixin message-mobile {
+  @include typography.style(body2);
+}
+
+@mixin actions {
+  display: grid;
+  justify-content: center;
+  grid-auto-flow: column;
+  gap: #{token(actions-spacing)};
+}
+
+@mixin actions-mobile {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  align-items: center;
```

Comment:
We can remove all of these styles. The only important one is `grid-auto-flow: row` now that the `.actions` container element is using a grid layout.